x-pack/filebeat/module/gsuite/saml/config/pipeline.js

```diff
@@ -0,0 +1,56 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+var saml = (function () {
+    var processor = require("processor");
+
+    var categorizeEvent = function(evt) {
+        evt.Put("event.type", "access");
```

Comment:
From ECS docs
```
Expected event types for category authentication:

start, end, info
```
I'm wondering if event.type should be info.

Reply:
Since those are login attempts, `access` made sense to me, any specific reason you think `info` might be a better fit here? `event.type` is defined as an array, you think might make sense to have both values maybe?

Reply:
@marc-gr so this is actually the perfect use case for the `event.outcome` field that you filled in just below. If it's a login attempt this should be `event.category = authentication, event.type = start` with `event.outcome` being either `success` or `failure` depending on whether the person was able to log in or not.

Reply:
I'm open to ["access", "info"].  My comment was motivated by the expected types being "start", "end" & "info".  start & end don't really fit since the event covers the whole login attempt, so I was thinking "info".  But I can see how "access" fits as well.  Maybe we should look at adding "access" to list of expected types?

Reply:
I looked into `event.type` docs without realising each category scoped the expected types for it in `event.category` ones and that your quote was from those 🤦 , now I see what you meant at the beginning, makes sense :+1: